### DISCUSSION TITLE
Proxy support

### DISCUSSION THREAD

**Discussion by TBDwarf** [tutao/tutanota#5648] [Ideas]
One idea, it would be interesting to add proxy support to Tutanota desktop.
Typically, at my company, because of their internal proxy and network policy, I can't use the desktop application... the ability to enter a proxy manually would solve the problem. So I'm forced to check my e-mails from Firefox.
The use of a proxy would also unblock access to Tutanota for users subject to censorship.

**Comment by newptcai** [comment]
Same here. Without a proxy, I cannot use Tutanota desktop.
